src/sfCommand.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Command, HelpSection, Interfaces } from '@oclif/core';
+
+export interface SfCommandInterface extends Interfaces.Command {
+  configurationVariablesSection?: HelpSection;
+  envVariablesSection?: HelpSection;
+  errorCodes?: HelpSection;
+}
+
+/**
+ * A base command that provides convenient access to common SFDX flags, a logger,
+ * CLI output formatting, scratch orgs, and devhubs.  Extend this command and set
+ * various static properties and a flag configuration to add SFDX behavior.
```

Comment:
This comment doesn't apply to the `SfCommand` class, right? Maybe something more like this?
```suggestion
 * A base command that provides convenient access to CLI help 
 * output formatting. Extend this command and set specific properties
 * to add help sections to the command's help output.
```